### ISSUE TITLE
Some brief documentation would be nice

### ISSUE THREAD

**Issue by thepurpleblob** [jquery-ui-bootstrap/jquery-ui-bootstrap#129]
Unless I'm being stupid, there's no documentation for this work. I'm sure I'll figure it out in the end but not being a hardcore jquery-ui user it isn't immediately obvious how to integrate it or, indeed, what the requirements are.

**Comment by gtraxx** [comment]
Hi, the index.html file includes examples of integration :).

The theme is in the folder: custom-theme

``` html
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>jQuery UI Bootstrap</title>
    <link type="text/css" href="css/custom-theme/jquery-ui-1.9.2.custom.css" rel="stylesheet" />
</head>
<body>
<p>Your content</p>
<script type="text/javascript" src="js/jquery-1.8.3.min.js"></script>
<script type="text/javascript" src="js/jquery-ui-1.9.2.custom.min.js"></script>
</body>
</html>
```

Best regards

**Comment by addyosmani** [comment]
I agree that we should add some more clear documentation on how the project should be used. Even a few steps on the homepage would guide people in the right direction. Thanks for bringing this to our attention @thepurpleblob!

**Comment by gtraxx** [comment]
Hi @thepurpleblob ,
We will create a new documentation based on the Bootstrap website.
What are your most important information first ?
:D

**Comment by johnnyreilly** [comment]
Hi Guys,

I haven't commented on this before but I'd be very interested in documentation on how the jQuery UI datepicker works in the world of jquery-ui-bootstrap.  Just looking at the demo page I noticed that an inline datepicker is displayed.  I'm guessing this doesn't mean that inline is the only way the datepicker can be used in jquery-ui-bootstrap?  

For now I've deliberately avoided using jQuery UI's datepicker thinking it wouldn't work and using the datepicker by @eternicode / Stefan Petri ( https://github.com/eternicode/bootstrap-datepicker ).  Since I'm better aquainted with the jQuery UI datepicker it would be nice to be able to use it with Twitter Bootstrap...

**Comment by thepurpleblob** [comment]
Hi,
It doesn't need to be much more than "To use this just add the css file xxxxx, and the js files yyyyy and zzzz" (if that's the case). It just to give a bit confidence that you have the right idea. All this stuff is obvious once you know it ;-)

There are quite a lot of files in the package and it's a little daunting. What do I need and what don't I?

Hope this helps.

**Comment by addyosmani** [comment]
I can get a start on this if we can nail down a list of what we want to cover. So..
- How do I include jQuery UI Bootstrap and use it in my project?
- Do you use a custom version of Bootstrap that works with jQuery UI?
- How do I pick and choose what to use?
- What gotchas or limitations do I need to be aware of?

Anything else?

**Comment by gtraxx** [comment]
Hi Addy,
I'll do it!
This is a great idea :)

**Comment by addyosmani** [comment]
Wonderful. Thanks! :)

**Comment by addyosmani** [comment]
How do we feel about the current docs? I could probably expand them with some  HTML examples to demonstrate some of our custom classes.

**Comment by gtraxx** [comment]
Yes it is a good idea, I start with respecting the style tooltip bootstrap.
But is it possible to reproduce popover with jQuery UI ?

**Comment by addyosmani** [comment]
I think we might have trouble with the jQuery UI popover, but we can definitely improve existing docs.

**Comment by Heliosmaster** [comment]
This should be somewhat closed by #159
